File: ezSCORES/ezSCORES.Model/FavoriteCompetitions.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class FavoriteCompetitions
	{
		public int Id { get; set; }

		public int UserId { get; set; }

		public int CompetitionId { get; set; }
		public virtual Competitions? Competition { get; set; }
	}
}

File: ezSCORES/UI/ezscores_mobile/lib/models/favoriteCompetitions.dart
import 'package:ezscores_mobile/models/competitions.dart';
import 'package:json_annotation/json_annotation.dart';

part 'favoriteCompetitions.g.dart';

@JsonSerializable()
class FavoriteCompetitions {
	int? id;
	int? userId;
	int? competitionId;
  Competitions? competition;

	FavoriteCompetitions();

	factory FavoriteCompetitions.fromJson(Map<String, dynamic> json) => _$FavoriteCompetitionsFromJson(json);

	Map<String, dynamic> toJson() => _$FavoriteCompetitionsToJson(this);
}
